src/view/drag-handle/sensor/create-keyboard-sensor.js

```diff
@@ -161,53 +171,65 @@ export default ({
     }
 
     if (event.keyCode === keyCodes.arrowLeft) {
-      stopEvent(event);
+      event.preventDefault();
       executeBasedOnDirection({
         vertical: schedule.crossAxisMoveBackward,
         horizontal: schedule.moveBackward,
       });
     }
 
-    blockStandardKeyEvents(event);
-
-    // blocking scroll jumping at this time
+    // preventing scroll jumping at this time
     if (scrollJumpKeys[event.keyCode]) {
-      stopEvent(event);
+      event.preventDefault();
+      return;
     }
+
+    preventStandardKeyEvents(event);
```

Comment:
Would it be worthwhile adding the keycodes that we're specifically handling above into this function as well so that we don't need to add `event.preventDefault` inside each conditional?

Reply:
these keys are ones that are prevented by standard across multiple sensors

Reply:
the keyboard sensor has its own unique keyboard controls